pkg/controllers/manila/manila.go

```diff
@@ -97,7 +97,7 @@ func (c *ManilaController) sync(ctx context.Context, syncCtx factory.SyncContext
 	shareTypes, err := c.openStackClient.GetShareTypes()
 	if err != nil {
 		switch err.(type) {
-		case *gophercloud.ErrEndpointNotFound:
+		case *gophercloud.ErrEndpointNotFound, gophercloud.ErrDefault403:
 			// OpenStack does not support manila, report the operator as disabled
 			return c.setDisabledCondition("This OpenStack cluster does not provide Manila service")
```

Comment:
Can you please set specific condition message, something like "Access to OpenStack Manila service denied", so we can distinguish NotFound/403? In addition, logging the error  could be useful.

Reply:
yes, will be done